### PR TITLE
docs/update-strategy: make airlock a link

### PR DESCRIPTION
**Pull request by nealey**

### CODE REVIEW

File: docs/usage/updates-strategy.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ In short, it consists of two operations:
  * unlock: after rebooting, any reboot slot owned by the node must be unlocked (and confirmed) by the lock-manager before proceeding further.
 
 This protocol is not coupled to any specific backend, and can be implemented on top of any suitable database.
-As an example, [airlock] is a free-software project which implements such protocol on top of [etcd3].
+As an example, [airlock](https://github.com/coreos/airlock) is a free-software project which implements such protocol on top of [etcd3](https://etcd.io/).
```
Comment (lucab) [inline docs/usage/updates-strategy.md:38]: It looks like my brain had an hiccup here. There are references at the bottom of this section, but I forgot to link them here (and the `airlock` one is plainly missing).

Can you please align with the existing bottom references?

Comment (nealey) [reply docs/usage/updates-strategy.md]: Today I learned a new way to make links in Markdown!

Comment (lucab) [reply docs/usage/updates-strategy.md]: A win-win PR! ;)

**Comment by lucab** [comment]
I've squashed this via GitHub merge logic. Thanks again for noticing and fixing this!

**Pull request merged**